src/main/java/ch/uzh/ifi/hase/soprafs24/model/game/Game.java

```diff
@@ -0,0 +1,45 @@
+package ch.uzh.ifi.hase.soprafs24.model.game;
+
+import ch.uzh.ifi.hase.soprafs24.constant.game.GameState;
+import ch.uzh.ifi.hase.soprafs24.entity.User;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+@Setter
+@Getter
+public class Game {
+    private Integer gameId;
+
+    private Long activePlayer;
+
+    private Long activePlayerStreak;
+
+    private GameState gameState;
+
+    private Long hostId;
+
+    private GameParameters gameParameters;
+
+    private List<Turn> history = new ArrayList<>();
+
+    private HashMap<String, Long> scoreBoard = new HashMap<>();
+
+    private List<User> players = new ArrayList<>();
+
+    private HashMap<String, Long> quickTurn = new HashMap<>();
```

Comment:
It's consistent with the Class Diagram. Nevertheless, we should use HashMap<Long, Long> in my opinion and reference userid instead of username (username is taken directly from spotify and not necessarily unique).